load_default_pseudomonas_data.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# Script to load default Pseudomonas data (in the files included in this
+# repository) into the adage-server database via management commands.
+
+# Import Pseudomonas annotation data
+python manage.py import_data ../data/PseudomonasAnnotation.tsv
+
+# Create Pseudomonas object in database for django-organisms
+python manage.py organisms_create_or_update \
+    --taxonomy_id=208964 \
+    --scientific_name="Pseudomonas aeruginosa" \
+    --common_name="Pseudomonas aeruginosa"
+
+# Add new machine learning model into database
+python manage.py add_ml_model "Ensemble ADAGE 300" 208964 --g2g_edge_cutoff 0.4
+
+# Import activity data into the database:
+python manage.py import_activity \
+    ../data/all-pseudomonas-gene-normalized_HWActivity_perGene_with_net300_100models_k=300_seed=123_ClusterByweighted_avgweight_network_ADAGE.txt \
+    "Ensemble ADAGE 300"
+
+# Save Ensembl and PseudoCAP Cross-reference Databases
+python manage.py genes_add_xrdb \
+    --name=Ensembl \
+    --URL=http://www.ensembl.org/Gene/Summary?g=_REPL_
```

Comment:
Aha. We do not have this step in our `fab` deploy script (just the one for PseudoCAP below). Do we need it?

Reply:
Ah, good catch - I think we don't actually need this here, and I put this in by accident. I will remove it.